script.js

```diff
@@ -1,69 +1,82 @@
+const titleInput = document.getElementById('book');
+const authorInput = document.getElementById('author');
 const form = document.getElementById('form');
-const inputBook = document.getElementById('book');
-const inputAuthor = document.getElementById('author');
-const Container = document.getElementById('container');
-const storedBookLocal = [];
+const container = document.getElementById('container');
+const bookskey = 'books';
+let deleteBtns = document.querySelectorAll('#remove');
 
-// Add Books
-
-const addBook = () => {
-  form.addEventListener('submit', (e) => {
-    e.preventDefault();
-
-    const authorbook = inputAuthor.value;
-    const titlebook = inputBook.value;
-    const itemlist = document.createElement('div');
-    itemlist.className = 'Itemlist';
-
-    const obj = {
-      Title: titlebook,
-      Author: authorbook,
-    };
+class Book {
+  constructor(title, author, id) {
+    this.author = author;
+    this.title = title;
+    if (id) {
+      this.id = id;
+    } else {
+      this.id = 1;
+    }
+  }
 
-    const book = `
-    <ul class=''>
-    <li>${obj.Title}</li>
-    <li>${obj.Author}</li>
-    </ul>
+  AddBook() {
+    const book = document.createElement('li');
+    book.innerHTML = `
+    <div class="description">
+    <h2>${this.title}</h2>
+    by
+    <h3>${this.author}</h3>
+    </div>
+    <button data-id=${this.id} id='remove'>Remove</button>
     `;
-    const removeBtn = document.createElement('button');
-    removeBtn.className = 'remove';
-    removeBtn.id = 'removeBtn';
-    removeBtn.innerText = 'Remove';
-
-    itemlist.innerHTML = book;
-    itemlist.appendChild(removeBtn);
-    Container.append(itemlist);
-
-    storedBookLocal.push(obj);
-    const result = JSON.stringify(storedBookLocal);
-
-    localStorage.setItem('data', result);
-  });
-};
-
-addBook();
+    return book;
+  }
+}
 
-const getDataStored = JSON.parse(localStorage.getItem('data'));
+const removeBook = (deleteBtns) => {
```

Comment:
Since I could not add book, I could not test remove book. Please ensure it works as expected